src/lib/engine/error.ts

```diff
@@ -147,7 +157,7 @@ function findErrorLineCol(stack: string | undefined): [number | null, number | n
 
 	if (location) {
 		let lineCol = location[1]!.split(":").map(Number)
-		line = lineCol[0]! - 2 - 1
+		line = lineCol[0]! - 2
```

Comment:
It'd be great eventually to extract this into a #define (or equivilent in this world), and give it an actual name (line offset for <x>)